### ISSUE TITLE
Build with Gulp not Make 

### ISSUE THREAD

**Issue by ponytrouble** [acdlite/flummox#157]
Novice adoption question: the layout is very useful with readme's fetched for a layout but, I'm unable to trial because of the make step.  Is Gulp or GRunt possible or, any suggestion on a equivalent React-Flummox app?

**Comment by acdlite** [comment]
Yeah I've gotten several requests for this. I'm not a huge fan of Gulp or Grunt, but to be accommodating I probably will convert it to Gulp soon.

**Comment by fczuardi** [comment]
consider using plain npm scripts instead of gulp: http://blog.keithcirkel.co.uk/how-to-use-npm-as-a-build-tool/ or just keep using make which is a very nice option as well

**Comment by tappleby** [comment]
yeah make is nice, doesn't work well for windows users though.
